### PR TITLE
RET 5188

### PR DESCRIPTION
**Pull request by LahiruOshara**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/RET-5084
https://tools.hmcts.net/jira/browse/RET-5191
https://tools.hmcts.net/jira/browse/RET-5192
https://tools.hmcts.net/jira/browse/RET-5081

### Change description ###
configurations related to claimant representative applications.
application for withdraw claimant is configured here with rule 92.

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent-ET1Repped.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   {"CaseTypeId": "ET_EnglandWales", "CaseEventID": "et3ResponseDetails", "UserRole": "[CLAIMANTSOLICITOR]", "CRUD": "D"},
   {"CaseTypeId": "ET_EnglandWales", "CaseEventID": "submitEt3", "UserRole": "[CLAIMANTSOLICITOR]", "CRUD": "D"},
   {"CaseTypeId": "ET_EnglandWales", "CaseEventID": "respondentTSE", "UserRole": "[CLAIMANTSOLICITOR]", "CRUD": "D"},
+  {"CaseTypeId": "ET_EnglandWales", "CaseEventID": "claimantTSE", "UserRole": "[CLAIMANTSOLICITOR]", "CRUD": "D"},
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent-ET1Repped.json:19]: Should this not be CRUD?
A claimant solicitor should have access to this event?

File: definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -1235,6 +1235,12 @@
     "UserRole": "caseworker-employment-legalrep-solicitor",
     "CRUD": "CRU"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseEventID": "claimantTSE",
+    "UserRole": "caseworker-employment-legalrep-solicitor",
+    "CRUD": "CRU"
+  },
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:1243]: I think this should be D?
A legal rep should not be able to access the claimantTSE event unless they have the claimantsolitictor role?

File: definitions/json/CaseEvent/CaseEvent.json [hunk 3]
```diff
@@ -1032,5 +1032,21 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "N"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "claimantTSE",
+    "Name": "Make an Application",
+    "Description": "Claimant Tell Something Else",
+    "PreConditionState(s)": "*",
```
[the remark this answers is not in the document]
Comment (LahiruOshara) [reply definitions/json/CaseEvent/CaseEvent.json]: resolved in https://github.com/hmcts/et-ccd-definitions-englandwales/pull/946/commits/4f772d528784427c0849821e05d607efda577c1a

File: definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json (hunk 2 above)
Comment (LahiruOshara) [reply definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json]: resolved in https://github.com/hmcts/et-ccd-definitions-englandwales/pull/946/commits/4f772d528784427c0849821e05d607efda577c1a

File: definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent-ET1Repped.json (hunk 1 above)
Comment (LahiruOshara) [reply definitions/json/AuthorisationCaseEvent/AuthorisationCaseEvent-ET1Repped.json]: resolved in https://github.com/hmcts/et-ccd-definitions-englandwales/pull/946/commits/4f772d528784427c0849821e05d607efda577c1a

File: definitions/json/CaseEvent/CaseEvent.json [hunk 4]
```diff
@@ -1032,5 +1032,21 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "N"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "claimantTSE",
+    "Name": "Make an Application",
+    "Description": "Claimant Tell Something Else",
+    "PreConditionState(s)": "Accepted",
```
Comment (hsjhita) [inline definitions/json/CaseEvent/CaseEvent.json:1041]:
```suggestion
    "PreConditionState(s)": "Submitted, Vetted, Rejected, Accepted",
```

File: definitions/json/CaseEvent/CaseEvent.json [hunk 5]
```diff
@@ -1032,5 +1032,21 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "N"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "claimantTSE",
+    "Name": "Make an Application",
+    "Description": "Claimant Tell Something Else",
+    "PreConditionState(s)": "Submitted, Vetted, Rejected, Accepted",
```
Comment (hsjhita) [inline definitions/json/CaseEvent/CaseEvent.json:1041]:
```suggestion
    "PreConditionState(s)": "Submitted;Vetted;Rejected;Accepted",
```

**Review by hsjhita — APPROVED** [review]

**Pull request merged**